app/src/main/java/com/example/imageEditor/MainActivity.kt

```diff
@@ -1,11 +1,29 @@
 package com.example.imageEditor
 
-import android.os.Bundle
-import androidx.appcompat.app.AppCompatActivity
+import android.util.Log
+import com.example.imageEditor.base.BaseActivity
+import com.example.imageEditor.databinding.ActivityMainBinding
+import com.example.imageEditor.model.CollectionModel
+import com.example.imageEditor.repository.HomeRepository
+import com.example.imageEditor.ui.HomeContract
+import com.example.imageEditor.ui.HomePresenter
 
-class MainActivity : AppCompatActivity() {
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+class MainActivity : BaseActivity<ActivityMainBinding>(), HomeContract.View {
```

Comment:
```suggestion
class MainActivity : BaseActivity<ActivityMainBinding>(), MainContract.View {
```
Nên đặt giống nhau cho dễ maintain sau này